joplin/api/schema.py

```diff
@@ -108,7 +108,7 @@ def resolve_related(self, resolve_info, *args, **kwargs):
 
     class Meta:
         model = ServicePage
-        filter_fields = ['id', 'slug', 'topic', 'topic__slug']
+        filter_fields = ['id', 'slug', 'topic', 'topic__slug', 'live']
```

Comment:
Hey if we limit the request to just return live pages all the time we won't have to make updates to the Janis repo. we could do this by adding to the Query class in this file.
```
def resolve_all_service_pages(self, info):
        return ServicePage.objects.live()

def resolve_all_processes(self, info):
        return ProcessPage.objects.live()
...
```
etc for all models?